app/controllers/comment_controller.rb

```diff
@@ -0,0 +1,9 @@
+class CommentsController < ApplicationController
+  def create
+    post = Post.find(params[:id])
+    comment = current_user.comments.new(text: params[:text])
+    comment.post = post
+    comment.save!
```

Comment:
- [ ] Kindly add a new method to instantiate the comments. This should read:
```ruby
def new
   @post = Comment.new
end
```
- [ ] Kindly add a success path in case the comment creation is successful and a failure path in case of failure
- [ ] Kindly use strong params when creating a new comment. This is the same as the private `post_params` method used in the posts controller.
- [ ] Kindly add a form for creating a new comment in the corresponding view.